### PR TITLE
initial attempt at Django web interface

### PR DESCRIPTION
**Pull request by whoburg**
work in progress, not ready for review

### CODE REVIEW

**Comment by whoburg** [comment]
made a lot of progress yesterday -- this is getting close. Still need to add a couple features, then this PR will be ready for review. Once we merge this pull request we should be ready to attempt deploying, and can make further improvements on new, smaller PR's.

**Comment by whoburg** [comment]
@matthewdominick , let's get this merged. This is ready for review.

File: feedbacksite/templates/registration/login.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <td>{{ form.username.label_tag }}</td>
+    <td>{{ form.username }}</td>
+</tr>
+<tr>
+    <td>{{ form.password.label_tag }}</td>
+    <td>{{ form.password }}</td>
+</tr>
+</table>
+
+<input type="submit" value="login">
```
Comment (matthewdominick) [inline feedbacksite/templates/registration/login.html]: Really?

Comment (matthewdominick) [reply feedbacksite/templates/registration/login.html]: (test comment)

**Pull request merged**